optuna/samplers/_tpe/sampler.py

```diff
@@ -145,8 +145,9 @@ def objective(trial):
                 versions without prior notice. See
                 https://github.com/optuna/optuna/releases/tag/v2.2.0.
         warn_independent_sampling:
-            If this is :obj:`True`, a warning message is emitted when
+            If this is :obj:`True` and `multivariate=True`, a warning message is emitted when
```

Comment:
A nit, but a pair of single backticks corresponds to italic.
```suggestion
            If this is :obj:`True` and ``multivariate=True``, a warning message is emitted when
```